ZenPacks/zenoss/LinuxMonitor/zenpack.yaml

```diff
@@ -200,6 +250,39 @@ device_classes: !ZenPackSpec
                                 dpName: volume_free
                                 lineType: AREA
                                 rpn: "size,-,size,/,-100,*"
+                    Physical Volume IO Operations Total:
```

Comment:
There's no need to have a *Physical Volume* prefix on all of the graph names. "Utilization" instead of "Physical Volume Utilization" would be cleaner.